<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtCMBMesherDialog</class>
 <widget class="QDialog" name="qtCMBMesherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meshing Dialog</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Mesher</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Mesher">
        <item>
         <property name="text">
          <string>Internal Mesher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Choose Mesher</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="FileNameText">
        <property name="toolTip">
         <string comment="File Name for the Input to the Mesher.">File name to save the Input to the Mesher.</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="AdvanceOptions">
        <property name="title">
         <string>Advance Options</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QFrame" name="AdvanceFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Preservation Mode</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QRadioButton" name="PreserveExterior">
                 <property name="toolTip">
                  <string>The exterior boundary of the input surface mesh will not be modified during volume meshing.</string>
                 </property>
                 <property name="text">
                  <string>Exterior Surface Mesh</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="PreserveSurface">
                 <property name="toolTip">
                  <string>The input surface mesh will not be modified during volume meshing.</string>
                 </property>
                 <property name="text">
                  <string>Entire Surface Mesh</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="PreserveNone">
                 <property name="toolTip">
                  <string>The volume mesher is free to modify any part of the input surface mesh.</string>
                 </property>
                 <property name="text">
                  <string>None</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="TetGenCommand">
              <property name="title">
               <string>Enter Tetgen Command Line Options</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="TetGenCommands"/>
               </item>
               <item>
                <widget class="QCheckBox" name="CommandLineOnly">
                 <property name="text">
                  <string>Use Command Line Options Only</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Mesh Length Size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="FileBrowserButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output Basename:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="MeshLength"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="MeshLengthType">
        <property name="toolTip">
         <string>Indicates if the length is an absolute size or a percentage of the smallest non-zero length of the Model's Bound</string>
        </property>
        <item>
         <property name="text">
          <string>Absolute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relative</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qtCMBMesherDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qtCMBMesherDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
